### ISSUE TITLE
testExecutableArgSingleCaseRegex is ignored

### ISSUE THREAD

**Issue by NovaNekmit** [Florin-Popescu/vscode-unity-test-adapter#15]
It was added to config and readme in 3e8308839bd08e8f058d8512e74824b497e7cbb2, but is never used.

Should be used here, but isn't: https://github.com/Florin-Popescu/vscode-unity-test-adapter/blob/master/src/adapter.ts#L427

Seems like a simple change, but I don't quite understand how it would get the test case name, as many of the functions are called with suite info only.

**Comment by Florin-Popescu** [comment]
Thanks for finding this. Looks like I completely forgot to follow through on that change. Planning to get this working around the holiday period.

**Comment by canton7** [comment]
Bump, just in case this has fallen off your radar

**Comment by ManuBoss55** [comment]
still no news?

**Comment by canton7** [comment]
Might be time for a fork, heh

**Comment by Florin-Popescu** [comment]
Hello!

I'm starting work on #12 and trying to understand what has changed in vscode APIs in the last 2 years since I've touched this extension. I will take a look at this issue afterwards since for now I'm not even sure whether the extension works any more.

I cannot say yet how long that will take since #12 may be pretty much a rewrite of the entire extension and I have very limited time. Any help would be appreciated!

**Comment by canton7** [comment]
It still works - I'm using it!

**Comment by Florin-Popescu** [comment]
Implemented this as part of the switch to native vscode API.

I thought that there's starting to be too many settings for this extension so I made the behavior hard coded instead of configurable. It will always pass `-n testName` to the test executable when a single test case (not a file) is run from the test interface. That will be replaced by `-g testGroup -n testName` if using the Fixtures test format.
